RNA was extracted using the Qiagen RNeasy Plant Mini Kit following the Qiagen protocol. Samples were sent to the Georgia Genomics and Bioinformatics Core (GGBC) for sequencing using Illumina Novaseq X Next Generation Sequencing (NGS). Quality checks were performed before and after quality control steps using FastQC and MultiQC. Quality control was performed using Trimmomatic v-0.39 to discard poor-quality sequences and adapters. Ribosomal RNA was removed using Ribodetector v-0.2.7. The trimmed sequences were mapped to the Joint Genome Institute Fragilariopsis cylindrus CCMP 1102 genome (Mock et al. 2017) using STAR v-2.7.11b. STAR GFF output files were converted to GTF files using the Bioconductor package GFFread v-0.12.7, and the subsequent GTF files were used with HTseq v-2.0.7 to generate read counts. Counts were normalized using the VST normalization within DESeq2 v-1.49.4, and a default differential expression analysis was performed for each pairwise grouping of treatments. Genome annotations were obtained from the JGI database for KOG, Gene Ontology (GO), EC pathways, Domains, and Pfams. The Bioconductor package KEGGREST v-1.0.1 was used to convert EC numbers into KO annotation information.  
